src/System.Numerics.Vectors/tests/Matrix3x2Tests.cs

```diff
@@ -1009,7 +1009,7 @@ struct Matrix3x2PlusFloat_2x
 
         // A test to make sure the fields are laid out how we expect
         [Fact]
-        [ActiveIssue(1002)]
+        [ActiveIssue("NullReferenceException")]
```

Comment:
@chcosta keeping this PR open is blocking the mirroring. Once you make your changes internally please close this PR so the mirror will pick-up the latest changes.